CONTRIBUTING.md

```diff
@@ -11,49 +10,51 @@ Thank you for your interest in contributing to Azure SDK for C.
+- **DO NOT** mix independent, unrelated changes in one PR. Separate real product/test code changes from larger code formatting/dead code removal changes. Separate unrelated fixes into separate PRs, especially if they are in different assemblies.
 
-Merging Pull Requests (for project contributors with write access)
-----------------------------------------------------------
+### Merging Pull Requests (for project contributors with write access)
 
-* **DO** use ["Squash and Merge"](https://github.com/blog/2141-squash-your-commits) by default for individual contributions unless requested by the PR author.
+- **DO** use ["Squash and Merge"](https://github.com/blog/2141-squash-your-commits) by default for individual contributions unless requested by the PR author.
   Do so, even if the PR contains only one commit. It creates a simpler history than "Create a Merge Commit".
   Reasons that PR authors may request "Merge and Commit" may include (but are not limited to):
 
   - The change is easier to understand as a series of focused commits. Each commit in the series must be buildable so as not to break `git bisect`.
   - Contributor is using an e-mail address other than the primary GitHub address and wants that preserved in the history. Contributor must be willing to squash
     the commits manually before acceptance.
 
-
-
 ## Developer Guide
 
 ### Prerequisites
-- CMake version 3.10 is required to use these libraries.
-- C compiler. MSVC, gcc or clang are recommended.
-- cmocka. For building and running unit tests. By default building unit tests is disabled, so, unless you want to add unit test or run it, you don't need to install this. See below how vcpkg can be used to install dependencies
-- curl. Curl is used a http stack and it is required for building and running service samples (keyvault and storage). You don't need to install curl if not building samples.
+
+- [CMake](https://cmake.org/download/) version 3.10 or later
+- C compiler: [MSVC](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019), [gcc](https://gcc.gnu.org/) or [clang](https://clang.llvm.org/) are recommended
+- [git](https://git-scm.com/downloads) to clone our Azure SDK repository with the desired tag
+- [cmocka](https://cmocka.org/) for building and running unit tests. By default, building unit tests is disabled, so, unless you want to add unit tests or run then, you don't need to install this. See how `vcpkg` can be used to install dependencies, [below](#VCPKG).
+- [curl](https://curl.haxx.se/download.html) which is used as an http stack, and is required for building and running service samples (keyvault and storage). You don't need to install curl if you are not building samples.
+- [doxygen](http://www.doxygen.nl/download.html) if you need to generate and view documentation.
 
 ### Development Environment
+
 Project contains files to work on Windows, Mac or Linux based OS.
 
 ### VCPKG
+
```

Comment:
I want to defer making substantial content cleanup to `CONTRIBUTING.md` to a separate PR (and I will probably re-write parts of it to be clearer/easier to read and follow).

That said, the topic of `vcpkg` in this context is just a mechanism/tool for contributors to get all the necessary dependencies they need to build/run tests and samples and contribute to the repo (such as curl, cmocka, etc.). What do you mean by "we don't support this"? There is no mention of support/how the SDK is released and consumed here.

Reply:
If you want to postpone, that's fine. 
But, since we don't use VCPKG, I don't think we have to explain it. I think just linking to where the prerequisites are is good enough. I want to focus on stuff specific to us and avoid the slippery slope that is everything else.  For example, I don't think we want the Visual Studio stuff in the document (as we discussed).